src/components/EndGameWindow.vue

```diff
@@ -0,0 +1,67 @@
+<script lang="ts" setup>
+import { useLobby } from '@/composables/useLobby'
+import { Views } from '@/types'
+import { Ref, ref } from 'vue'
+import { useRouter } from 'vue-router'
+
+const { lobby, isLobbyReady } = useLobby()
+const router = useRouter()
+
+const result = ref<string>('')
```

Comment:
Czytam też ostanio fajną kisążke o TS "TypeScript na poważnie - Michał Miszczyszyn" (polecanko przy okazji) i tutaj nie wiem co Ty uważasz ale ja mogłem Cię źle pokierować. Zmieniłem zdanie co do typowania wszystkiego co się da na rzecz z jak największego korzystania z inferencji typów.

W tym projekcie napewno już nie będziemy tego zmieniać ale inferencja dla primitive i return type funkcji gdy nie zwracają one obiektów może zawsze być inferowana i fajnie to działa